coalib/bears/GlobalBear.py

```diff
@@ -4,22 +4,28 @@
 
 class GlobalBear(Bear):
     """
-    A GlobalBear is able to analyze semantic facts across several file.
+    A GlobalBear analyzes semantic facts across several files.
 
     The results of a GlobalBear will be presented grouped by the origin Bear.
-    Therefore Results spanning above multiple files are allowed and will be
-    handled right.
+    Therefore Results spanning across multiple files are allowed and will be
+    handled correctly.
 
-    If you only look at one file at once anyway a LocalBear is better for your
-    needs. (And better for performance and usability for both user and
-    developer.)
+    If you are inspecting a single file at a time, you should consider
+    using a LocalBear.
     """
 
     def __init__(self,
-                 file_dict,  # filename : file contents
+                 file_dict,
                  section,
                  message_queue,
                  timeout=0):
+        """
+        Constructs a new GlobalBear.
+
+        :param file_dict: The dictionary of {filename: file contents}.
+
```

Comment:
> Duplication of documentation isn't bad at all

Disagree. When changing documentation, you have to track down all the duplicated places. Duplication is always bad, be it either code or docs.

> It has been intentionally done in some modules as well (see Linter.py for example).

I have taken a look at that 880 lines file. Indeed, there is some duplicated documentation. However, there is some "metaclass conflict" which is probably the root cause of that (I have no idea though). So, please either be more specific and/or let's hear a third opinion about duplication.